.github/workflows/CI.yml

```diff
@@ -15,46 +15,44 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1
+          - "1.0"  # LTS
```

Comment:
Would we like to explicitly test on 1.5 since we're using it internally?

Reply:
https://github.com/invenia/TrackedDistributions.jl/pull/9/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R36